console-subscriber/src/aggregator/mod.rs

```diff
@@ -320,19 +322,36 @@ impl Aggregator {
             let (tx, rx) = mpsc::channel(buffer);
             let subscription = Watch(tx);
             let now = Some(self.base_time.to_timestamp(Instant::now()));
+
+            let span = id.clone();
+            // XXX(jswrenn): A new thread is spawned solely for the sake of
+            // getting access to the global subscriber.
+            let (trace, updates) = std::thread::spawn(move || {
+                // XXX(jswrenn): Need to gracefully handle the case where the
+                // task span is already closed.
+                consequences::trace(&span, buffer)
+                    .expect("the subscriber isn't a Registry")
+                    .expect("id is already closed")
+            })
+            .join()
+            .unwrap();
```

Comment:
this is quite unfortunate, i'd like to figure out a better solution for this. at the very least, we could probably do this a _single_ time when the aggregator task is spawned, and clone the `Dispatch`, so that we don't have to do this every subsequent time we want to get details for a task...